expat/lib/xmlparse.c

```diff
@@ -5391,6 +5432,13 @@ processInternalEntity(XML_Parser parser, ENTITY *entity,
   enum XML_Error result;
   OPEN_INTERNAL_ENTITY *openEntity;
 
+  if (!(parser->m_options & XML_OPTION_HUGE_ENTITES)) {
+      parser->m_entityRecursionLevel++;
+      if (parser->m_entityRecursionLevel >= XML_ENTITY_RECURSION_LIMIT) {
```

Comment:
I noticed that `XML_ENTITY_RECURSION_LIMIT` is compared with "greater than or equal to" while `XML_ENTITY_EXPANSION_SIZE` is compared with "greater than". Is that difference intentional?